src/apis/SubredditsByPost.es6.js

```diff
@@ -0,0 +1,25 @@
+import { isEmpty } from 'lodash/lang';
```

Comment:
🔕  In general the best practice for importing this is:

```javascript
import isEmpty from 'lodash/isEmpty';
```

We should keep it as is for consistency and take a cleanup pass soon.